javascript/javascript2/week1/homework/findShortestWord/index.js

```diff
@@ -0,0 +1,45 @@
+//Find the shortest word 
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function getShorterWord(danishWords){ 
+    const lengthOfDanishWords = [];
+    danishWords.forEach(word => {
+    lengthOfDanishWords.push(word.length)
+});
+    let shorterDanishWord = Math.min.apply(Math, lengthOfDanishWords);
+    danishWords.forEach(word => {
+       if (word.length == shorterDanishWord){
+        console.log(`Shortest Danish word : ${word}`);
+       }
+    });
```

Comment:
Consistent formatting is important, do a search for `format code in visual studio code` to see how Visual Studio Code can format your code consistently.

Reply:
Thank you for spotting with it. I always check my code. I missed this one!

Reply:
I deleted the previous comment by mistake instead of the file!